portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,13 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+  
 }
+async function showServerTime() {
```

Comment:
[4.6.1](https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace): A single blank line appears Between consecutive methods

```suggestion
}

async function showServerTime() {
```